src/linear_algebra/pi_tensor_product.lean

```diff
@@ -148,18 +135,10 @@ lemma add_tprod_coeff' (z₁ z₂ : R) (f : Π i, s i) :
   tprod_coeff R z₁ f + tprod_coeff R z₂ f = tprod_coeff R (z₁ + z₂) f :=
 quotient.sound' $ add_con_gen.rel.of _ _ (eqv.of_add_scalar z₁ z₂ f)
 
-lemma add_tprod (f : Π i, s i) (i : ι) (m₁ m₂ : s i) :
-  (⨂ₜ[R] j, (update f i m₁) j) + (⨂ₜ[R] j, (update f i m₂) j) = ⨂ₜ[R] j, (update f i (m₁ + m₂)) j :=
-add_tprod_coeff 1 f i m₁ m₂
-
 lemma smul_tprod_coeff (z : R) (f : Π i, s i) (i : ι) (r : R) :
   tprod_coeff R z (update f i (r • f i)) = tprod_coeff R (r * z) f :=
 quotient.sound' $ add_con_gen.rel.of _ _ $ eqv.of_smul _ _ _ _
 
-lemma smul_tprod_index (f : Π i, s i) (i j : ι) (r : R) :
```

Comment:
This lemma doesn't exist any more, but probably belongs somewhere in `multilinear_map` if its useful